src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -128,6 +128,12 @@ Copyright (c) .NET Foundation. All rights reserved.
                  FormatArguments="$(SdkResolverGlobalJsonPath)" />
   </Target>
 
+  <Target Name="_WarnWhenUsingNET6AndVSPriorTo17"
```

Comment:
We also need to make this only warn when targeting .NET 6 and higher.

Reply:
Added that condition 👍